<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormServicioBase</class>
 <widget class="QWidget" name="FormServicioBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LENombre"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Descripcion:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="TEDescripcion">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Fecha de Alta del servicio:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="DEFechaAlta">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="CkBBaja">
           <property name="text">
            <string>Dado de Baja</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="DEFechaBaja">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Datos del servicio</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Precio Base:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EDSBPrecio" name="dSBPrecioBase">
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="prefix">
             <string>$ </string>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>9999999.990000000223517</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Periodo de Cobro:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="CBPeriodo"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Dia en que se hace el cobro:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="CBInicioCobro"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Metodo de cobro incompleto:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="CBMetodoIncompleto"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EDSBPrecio</class>
   <extends>QDoubleSpinBox</extends>
   <header>edsbprecio.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
